### PR TITLE
improve wizard for marketplace

### PR DESCRIPTION
**Pull request by orthagh**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Follow | #6933 

After early presentation of the feature, we think about adding some additional steps to the marketplace wizard

### CODE REVIEW

**Comment by orthagh** [comment]
image

File: inc/marketplace/view.class.php [hunk 1]
```diff
@@ -150,7 +150,10 @@ static function checkRegister() {
             array_push(
                $messages,
                __('Your GLPI Network registration is not valid'),
-               "<a href='$config_url'>".__("Set your registration key")."</a>"
+               __('A registration, at least a free one, is required to use marketplace!'),
+               "<a href='".GLPI_NETWORK_SERVICES."'>".__('Register on GLPI Network!')."</a> ".
```
Comment (cedric-anne) [inline inc/marketplace/view.class.php:154]:
```suggestion
               "<a href='".GLPI_NETWORK_SERVICES."'>".__('Register on GLPI Network')."</a> ".
```

File: inc/marketplace/view.class.php [hunk 2]
```diff
@@ -150,7 +150,10 @@ static function checkRegister() {
             array_push(
                $messages,
                __('Your GLPI Network registration is not valid'),
```
Comment (cedric-anne) [inline inc/marketplace/view.class.php:152]:
```suggestion
               __('Your GLPI Network registration is not valid.'),
```

File: inc/marketplace/view.class.php [hunk 3]
```diff
@@ -150,7 +150,10 @@ static function checkRegister() {
...
-               "<a href='$config_url'>".__("Set your registration key")."</a>"
+               __('A registration, at least a free one, is required to use marketplace!'),
+               "<a href='".GLPI_NETWORK_SERVICES."'>".__('Register on GLPI Network!')."</a> ".
+               __('and'). " ".
+               "<a href='$config_url'>".__("fill your registration key in setup")."</a>"
```
Comment (cedric-anne) [inline inc/marketplace/view.class.php:156]:
```suggestion
               "<a href='$config_url'>".__("fill your registration key in setup.")."</a>"
```

**Pull request merged**